spec/dummy/app/models/time_record.rb

```diff
@@ -4,5 +4,22 @@ class TimeRecord < ApplicationRecord
   belongs_to :project
   has_one :workspace, through: :project
   has_and_belongs_to_many :tags, -> { distinct }
-  
+
+  scope :by_workspace, ->(workspace_id) { joins(:project).where("projects.workspace_id = ?", workspace_id) }
+
+  def staff?
```

Comment:
I think it should be deleted.

Reply:
why?
it's method for dummy class, we need have this for ability to stub that in controller